shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java

```diff
@@ -57,7 +57,13 @@ public interface PlatformPluginDelegate {
      */
     boolean popSystemNavigator();
 
-    void setFrameworkHandlesBack(boolean frameworkHandlesBacks);
+    /**
+     * The Flutter application would or would not like to handle navigation pop events itself.
+     *
+     * <p>Relevant for registering and unregistering the app's OnBackInvokedCallback for the
+     * Predictive Back feature.
+     */
+    default void setFrameworkHandlesBack(boolean frameworkHandlesBack) {}
```

Comment:
I think this works as a good alternative to the empty implementation in `FlutterFragment`. Maybe the doc can expand on where this is currently relevant (`FlutterActivity` alone now I suppose)?